node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -308,9 +352,14 @@ type OsDeviceConnectivityHelperInterface interface {
 		Mainly for writting clean unit testing, so we can Mock this interface in order to unit test OsDeviceConnectivityHelperGeneric logic.
 	*/
 	GetHostsIdByArrayIdentifier(arrayIdentifier string) ([]int, error)
-	GetDmsPath(volumeId string, volumeNguid string) (string, error)
+	GetDmsPath(identifiers []string) (string, error)
 	GetWwnByScsiInq(dev string) (string, error)
+	IsMpathMatchVolumeId(dmPath string, identifiers []string) (bool, error)
+	IsMpathMatchVolumeIdWithoutErrors(dmPath string, identifiers []string) bool
 	ReloadMultipath() error
+	GetMpathOutputByVolumeId(identifiers []string) (string, error)
```

Comment:
```suggestion
	GetDmsPath(idVariations []string) (string, error)
	GetWwnByScsiInq(dev string) (string, error)
	IsMpathMatchVolumeId(dmPath string, idVariations []string) (bool, error)
	IsMpathMatchVolumeIdWithoutErrors(dmPath string, idVariations []string) bool
	ReloadMultipath() error
	GetMpathOutputByVolumeId(idVariations []string) (string, error)
```
also:
`for _, identifier := range identifiers` -> `for _, id := range idVariations`, etc...

I know I originally suggested the name `identifiers` (which is better than having separate uuid+nguid parameters), but on second thought, I think it might be easily confused with `arrayIdentifiers`, and it's also doesn't explain their relation with each other (in contrast to "variations", which implies they represent the same "general" volume id).
sorry for the double work

Reply:
even better:
`idVariations` -> `volumeIdVariations`
@ArbelNathan what do you think?

Reply:
sounds good to me, took me a sec to realize that its not array identifiers even here in the suggestion..

Reply:
do you think that I need to change also [here](https://github.com/IBM/ibm-block-csi-driver/blob/f9f084833062cb7a610690393db475274311ae0c/node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go#L629)?

Reply:
done